dockrsync

```diff
@@ -194,22 +203,30 @@ function watch {
         anybar orange
         echo "$(date)" "\"$event\" changed. Synchronizing... "
         file="${event/$(dir)/$PROJECT_DIR_WITH_DOT}"
+
+        if [ -n "$RSYNC_DELETE_FLAG" ]; then
+          if [ ! -f $file ]; then
+            file="$(dirname "$file")"
```

Comment:
As rsync can't sync files which don't exists, I have synced the whole parent directory. I don't see any other option here, suggestions?